lib/index.ts

```diff
@@ -138,6 +137,12 @@ export function persistTree(structure : any, store : any, debug: boolean) {
              * we care about, which are defined in structure.
              */
             const newState = _pickBy(state, (value : any, key : any) => {
+                /**
+                 * If nothing is passed to the `values`
+                 * parameter, all values will be passed.
```

Comment:
Just a language thing here, could we change it to "all values will be **used**" ?